src/mono/mono/mini/CMakeLists.txt

```diff
@@ -554,3 +554,9 @@ if(NOT DISABLE_EXECUTABLES)
     install(FILES $<TARGET_PDB_FILE:mono-sgen> DESTINATION bin OPTIONAL)
   endif()
 endif()
+
+if(HOST_ALPINE_LINUX)
+  # On Alpine Linux, we need to ensure that the reported stack range for the primary thread is
```

Comment:
This should probably be moved to the main CMakeFiles.txt where the other if(HOST) definitions are.